client/globals.d.ts

```diff
@@ -10,6 +10,15 @@ declare const wcpaySettings: {
 	isJetpackIdcActive: boolean;
 	accountStatus: {
 		error?: boolean;
+		deposits?: {
+			completed_waiting_period: boolean;
+			delay_days: number;
```

Comment:
It looks like `delay_days` can also be sent as `null`, as per https://github.com/Automattic/woocommerce-payments/blob/57eb845ca3562595e847dcd199ba3a940f68e711/includes/class-wc-payments-account.php#L341

```suggestion
			delay_days: number | null;
```

Reply:
Fixed in 640ca5f.